Modules/_csv.c

```diff
@@ -45,9 +45,10 @@ _csv_free(void *m)
    _csv_clear((PyObject *)m);
 }
 
-static struct PyModuleDef _csvmodule;
+static struct PyModuleDef _csvmodule_def;
+static PyObject *_csvmodule;
```

Comment:
Do we need this at file scope? Given that this is replacing ``PyObject *module`` at line 1618

Reply:
`_csvmodule_def` is required in `_csvstate_global`. But you are right, we don't need _csvmodule at file scope. Will fix.